mkosi/config.py

```diff
@@ -1975,13 +1981,13 @@ def parse_new_includes(
         namespace: argparse.Namespace,
         defaults: argparse.Namespace,
     ) -> Iterator[None]:
-        l = len(getattr(namespace, "include", []))
+        current_includes = len(getattr(namespace, "include", []))
```

Comment:
This is a variable containing the length of the current includes so the variable name doesn't really seem right

Reply:
Changed the name. PTAL.